code/modules/VR/vr_human.dm

```diff
@@ -2,6 +2,7 @@
 
 /mob/living/carbon/human/virtual_reality
 	var/mob/living/carbon/human/real_me //The human controlling us, can be any human (including virtual ones... inception...)
+	var/datum/mind/real_mind
```

Comment:
in an ideal case, you would simply track the source mob from the mind, that way if the real mind gets mindswapped, or some other crazy shit happens, you will track damage the actual appropriate real mob

and yes that could happen if I am guessing right

Reply:
i.e mind.current